src/core/public/chrome/ui/header/_index.scss

```diff
@@ -29,19 +16,8 @@
   margin-right: $euiSize;
 }
 
-// Mobile header is smaller
-@include euiBreakpoint('xs', 's') {
-  .chrHeaderWrapper ~ .app-wrapper:not(.hidden-chrome) {
-    left: 0;
-  }
-}
-
-@include euiBreakpoint('xl') {
-  .chrHeaderWrapper--navIsLocked {
-    ~ .app-wrapper:not(.hidden-chrome) {
-      // Shrink the content from the left so it's no longer overlapped by the nav drawer (ALWAYS)
-      left: $euiNavDrawerWidthExpanded !important; // sass-lint:disable-line no-important
-      transition: left $euiAnimSpeedFast $euiAnimSlightResistance;
-    }
+.header__toggleNavButtonSection {
+  .euiBody--collapsibleNavIsDocked & {
+    display: none;
```

Comment:
Is this doing the same thing that the `showButtonIfDocked` prop does on `EuiCollapsibleNav`?

Reply:
Yes... If I was using the button supplied by `EuiCollapsibleNav`... Because of the stuff on the right of the header, I'm using my own button to keep focus order predictable. (Using the button supplied by EUI makes the focus order ToggleButton -> Nav down the left -> Header -> Rest of the page; With a custom button the order is Header -> Nav down the left -> Rest of the page.)

I don't think there's a way that EUI can do this for us and is a reasonable default for them but is flexible enough for Kibana.